src/coverlet.msbuild.tasks/CoverageResultTask.cs

```diff
@@ -22,7 +23,7 @@ public class CoverageResultTask : Task
         private string _thresholdStat;
         private string _coverletMultiTargetFrameworksCurrentTFM;
         private ITaskItem _instrumenterState;
-        private MSBuildLogger _logger;
+        private ILogger _logger;
```

Comment:
We can remove this...and use local one in `Execute`

Reply:
Done.